ajax/libs/bootstrap-horizon/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "bootstrap-horizon",
+  "filename": "bootstrap-horizon.min.css",
+  "version": "0.1.0",
+  "description": "Boostrap 3 rows with inline, horizontally scrolling columns",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/Zertz/bootstrap-horizon.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/Zertz/bootstrap-horizon.git",
+    "basePath": "",
+    "files": [
+      "bootstrap-horizon*"
+    ]
+  },
```

Comment:
@hjames9 We usually prefer to use npm auto-update if library been published on npm. Let's use npm auto-update here. Thanks.